pkg/kt/snmp.go

```diff
@@ -243,6 +243,7 @@ type SnmpDiscoConfig struct {
 	ReplaceDevices     bool            `yaml:"replace_devices"`
 	NoDedup            bool            `yaml:"no_dedup_engine_id,omitempty"`
 	CheckAll           bool            `yaml:"check_all_ips,omitempty"`
+	DoubleCheckV1      bool            `yaml:"doublecheck_snmp_v1,omitempty"`
```

Comment:
I think the specific flag name is tripping me up a bit. I feel like `doublecheck` might imply that something is literally being checked twice. Not sure about the best replacement name though. Maybe something like `check_all_snmp_versions`? That could be confusing if the user doesn't provide V3 info though. Maybe `use_snmp_v1_fallback`?

Reply:
I like `check_all_snmp_versions`.